### PR TITLE
Manager: For Statistics graph show values on Y axis with 2 digits aft…

### PR DESCRIPTION
**Pull request by AenBleidd**
…er point if these two digits are not zero only

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

**Comment by AenBleidd** [comment]
Fixed #656

File: clientgui/ViewStatistics.cpp [hunk 1]
```diff
@@ -739,8 +739,11 @@ void CPaintStatistics::DrawAxis(wxDC &dc, const double max_val_y, const double m
 	double y_start_val = ceil(min_val_y / d_oy_val) * d_oy_val;
 	d_oy_count = (int)floor((max_val_y - y_start_val) / d_oy_val);
 
+    double temp_int_part;
+    int prec = modf(d_oy_val, &temp_int_part) == 0.0 ? 0 : 2;
```
Comment (ChristianBeer) [inline clientgui/ViewStatistics.cpp]: How does this cope with values like `5.001`? Shouldn't that set precision to 0? Instead it is set to 2.

Comment (AenBleidd) [reply clientgui/ViewStatistics.cpp]: Ok. Will do

Comment (AenBleidd) [reply clientgui/ViewStatistics.cpp]: Fixed

**Comment by AenBleidd** [comment]
@davidpanderson, could you please review and merge this? Thanks

**Comment by davidpanderson** [comment]
Thanks!

**Pull request merged**